test/deploy.py

```diff
@@ -0,0 +1,569 @@
+#!/usr/bin/python3
+import os
+import sys
+import pwd
+import json
+import yaml
+import time
+import pylxd
+import string
+import random
+import subprocess
+
+
+def _get_random_string(length: int) -> string:
+    """Get a randomised string of given lentgh."""
+    return "".join(
+        random.choices(string.ascii_uppercase + string.digits, k=length)
+    )
+
+
+class Cleaner:
+    def __init__(self, model_file_path: string) -> None:
+        if os.path.exists(model_file_path):
+            # init client
+            self.client = pylxd.Client()
+            self.clean(model_file_path)
+        else:
+            print("Model File {} does not exists.".format(model_file_path))
```

Comment:
Why not use the logging module ? I understand this script is meant to run locally, but I find the logging interface to be more ergonomic when we need to redirect the output.

Reply:
The script stops execution as soon as it comes across an "unexpected" error. Using print was 1. simpler (for 1st iteration as least) and 2. the error (relevant for what caused the script to fail) always comes on the execution output last. I plan to add a multinode deployment cmd in future as well (deploys HA ceph on 3 lxd nodes) and initially planned to switch to logging in that PR.